channels/smartcard/client/smartcard_pack.c

```diff
@@ -2687,7 +2687,7 @@ LONG smartcard_pack_state_return(SMARTCARD_DEVICE* smartcard, wStream* s, const
 {
 	LONG status;
 	DWORD cbAtrLen = ret->cbAtrLen;
-	DWORD index = 0;
+	UINT32 index = 0;
```

Comment:
This should not be necessary, `UINT32` and `DWORD` are both defined to the same 32bit unsigned integer type, if not then something in winpr is missing for MinGW